nilearn/regions/tests/test_parcellations.py

```diff
@@ -45,7 +45,7 @@ def test_error_parcellation_method_none(test_image):
         Parcellations(method=None, verbose=0).fit(test_image)
 
 
-@pytest.mark.parametrize("method", ["kmens", "avg", "complte"])
+@pytest.mark.parametrize("method", ["kmens", "avg", "complete"])
```

Comment:
These are misspelled on purpose but you can change it to something that won't fail like "comp" or just configure to ignore it

Reply:
ha yes 
silly me
will fix